### ISSUE TITLE
indent syntax  in disambiguations

### ISSUE THREAD

**Issue by yanosh-igor** [spencermountain/wtf_wikipedia#166]
Hi, 
Check this article https://en.wikipedia.org/wiki/The_Cool_Kids

There is some helpful sentence at the beginning `For the upcoming television sitcom, see The Cool Kids (TV series).` But it is not related to the text of article. And article text starting from sentence `The Cool Kids is an American alternative hip hop duo..`
Will be good to move this sentence into some specific field or to the end of the text.

Snippet from dump:
```
{{good article}}
:''For the upcoming television sitcom, see [[The Cool Kids (TV series)]].''
{{Infobox musical artist..
```

**Comment by spencermountain** [comment]
oh hey, i'm not familiar with that `:` syntax for disambuations.

a leading :  indents the line, maybe we should strip leading indented sentences

for reference:
```
no indent (normal)
:first indent
::second indent
:::third indent
::::fourth indent
```

**Comment by spencermountain** [comment]
yeah, i stripped any indented first sentence
not sure if it will screw up some other stuff
we'll see
fixed in 5.3.0
